django/core/checks/registry.py

```diff
@@ -8,6 +8,7 @@ class Tags:
     Built-in tags for internal checks.
     """
     admin = 'admin'
+    async_ = 'async'
```

Comment:
This tag should be documented in the `docs/ref/checks.txt` (`Builtin tags`) with `versionchanged` annotation.

Reply:
Ah yeah.. thanks for mention it! Didn't aware of that before.

Btw, shouldn't we use `versionadded` instead of `versionchanged`?

Reply:
Builtin tags already exist, so `versionchanged`.